bashrc

```diff
@@ -108,7 +109,8 @@ export HISTFILESIZE=-1
 export HISTTIMEFORMAT="%Y-%m-%d %H:%M:%S "
 export HISTCONTROL=ignorespace
 export HISTIGNORE=ls:ps
-PROMPT_COMMAND="${PROMPT_COMMAND:+$PROMPT_COMMAND;}"'echo $$ "$(history 1)" >> "${HOME}"/.bash_history+/$(date +%Y%m%d)'
+#mlg# PROMPT_COMMAND="${PROMPT_COMMAND:+$PROMPT_COMMAND;}"'echo $$ "$(history 1)" >> "${HOME}"/.bash_history+/$(date +%Y%m%d)'
```

Comment:
No need to keep the previous line commented out.

Reply:
Ok! I will remove the commented line!